doc/website/images/iceoryx_components_diagram_nextgen_release.svg

```diff
@@ -1,47 +1,46 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We should create a CI job that updates the `.svg`'s.

@dkroenke What do you think?

Reply:
Sure, would you create a ticket for that?

Reply:
Done: https://github.com/eclipse-iceoryx/iceoryx/issues/1156